application.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Can you move this configuration file to the resources folder?

Reply:
Done